ergo/platforms/almanis.py

```diff
@@ -0,0 +1,226 @@
+from abc import abstractmethod
+import json
+from typing import Dict, List, Sequence
+
+import requests
+
+from ergo.questions import BinaryQuestion, Question
+
+
+class Almanis:
+    """
+    The main class for interacting with Almanis
+    """
+
+    question_id_request_json = json.dumps(
+        {
+            "customerID": "44d7f0b3-7e6f-4867-b00a-6816492bb510",
+            "statuses": [{"Trading": {}}, {"ClosedForTrading": {}}],
+        }
+    )
+
+    BINARY_QUESTION_KEY = "BooleanQuestionOD"
+    CONTINUOUS_QUESTION_KEY = "MultipleChoiceQuestionOD"
```

Comment:
Seems like there's a difference between multiple choice and continuous; multiple choice is >2 discrete choices, continuous is like any real value between 0 and 100.

Reply:
(Because of this, I also don't think it makes sense to try to use `PointDensity` to represent multiple-choice questions)

Reply:
Yeah, I mean in practice this representation is not "continuous" as it is a few concrete options, but I think the Almanis platform uses this type of question to ask something like "What will some continuous value be in the future" as opposed to "Will this thing happening, yes or no". For example an Almanis question (I can't link it because of how their web app works) asks about what the Dow will be at on 12/14, and [this Metaculus question](https://www.metaculus.com/questions/3472/what-will-teslas-market-capitalization-be-on-1-january-2021/) asks about Tesla's Market cap in 2021. Even though responses are represented differently, this seems like the same type of question to me. For example, in some world where the questions were phrased exactly the same (say both were asking about the Dow), we'd like to compare these two distributions as like objects. Am I making sense here?

Also, as I was poking around about this, there *are* some questions in Almanis that are true categorical multiple choice questions (e.g. "Which vaccine will be the most administered as at 12 June 2021?"), and those would obviously not qualify has the type of "continuous" question I'm talking about. I would have to make a new multiple choice class for those.